src/components/Routes.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Route, BrowserRouter as Router } from 'react-router-dom';
+import Login from './Login';
+import Landing from './Landing';
+
+const Routes = () => (
+  <>
+    <Router>
+      <Route exact path="/" component={Landing} />
+      <Route path="/login" component={Login} />
```

Comment:
There's no test file for Login component.

Reply:
On it...

Reply:
@adesege worked on feedback